build.sh

```diff
@@ -18,8 +18,8 @@
 
 WORKSPACE_DIR=
 THREADS=1
-CLEAN=false
-TEST=false
+CLEAN=true
+TEST=true
```

Comment:
ah didnt know that, but I did know not supposed to change the build file. Thanks for the tip